### PR TITLE
Add architecture terms to glossary

### PR DESCRIPTION
**Pull request by keirthana**

### CODE REVIEW

**Review by ajanon — APPROVED** [review]
LGTM!

File: ref/glossary.md [hunk 1]
```diff
@@ -322,6 +348,16 @@ A system container and virtual machine manager that offers a unified user experi
 
 See [the LXD website](https://linuxcontainers.org/).
 
+<a name="lxd-cluster"></a>
+### LXD cluster
+
+A set of LXD servers that share the same distributed database that holds the configuration for the cluster members and their instances.
```
Comment (adglkh) [inline ref/glossary.md:354]: LXD servers
->
LXD nodes

File: ref/glossary.md [hunk 2]
```diff
@@ -248,6 +269,11 @@ One of the main objects of Anbox Cloud. Every time you launch an application or
 
 See https://discourse.ubuntu.com/t/about-containers/17763.
 
+<a name="control-node"></a>
+### Control node
```
Comment (adglkh) [inline ref/glossary.md:273]: Is this `Control node` the management node that we described below? If so, let's be consistent in the way we use terms on this page.

File: ref/glossary.md [hunk 3]
```diff
@@ -135,6 +141,21 @@ A self-contained deployment variant of Anbox Cloud.
...
 
+<a name="anbox-cloud-cluster"></a>
+### Anbox Cloud cluster
+
+A deployment of the Anbox Cloud, either just the core stack or the core stack along with the streaming stack.
+
+<a name="anbox-cloud-subcluster"></a>
+### Anbox Cloud subcluster
+
+The group of components that is made up of the management layer (AMS, AMC, etcd), control node, LXD, AMS node controller.
```
Comment (adglkh) [inline ref/glossary.md:152]: I am not quite sure if we wanna mention the control node here next to `the management layer` as they seem to be the same to me.

**Review by adglkh — CHANGES_REQUESTED** [review]

**Review by adglkh — APPROVED** [review]
LGTM

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**